### PR TITLE
[PREVIEW] Create Respond step

### PR DESCRIPTION
**Pull request by borislavpetrovhmcts**
# Description
This creates Respond step.

Fixes # (issue) DIV-3100

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Locally,  unit, e2e.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -31,31 +32,50 @@ lookAndFeel.configure(app, {
...
+app.use('/public', (req, res) => {
+  res.redirect(req.path, '301');
+});
+// redirect images from reform-pattern-library
+app.use('/images', (req, res) => {
+  res.redirect(`/assets/images${req.path}`, '301');
+});
+
 onePerPage.journey(app, {
   baseUrl: config.node.baseUrl,
```
Comment (dtaylor7) [inline app.js]: dont need this line

File: common/content.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "en": {
-    "serviceName": "Service Name"
+    "serviceName": "Service Name",
```
Comment (dtaylor7) [inline common/content.json]: this isnt used anywhere and can be removed

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+redis:
```
Comment (dtaylor7) [inline docker-compose.yml]: dont comit compose file - add this to git ignore

**Comment by borislavpetrovhmcts** [comment]
Good spot! All fixed

**Pull request merged**